src/server/test/util/unitTestUtils.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+const { chai, mocha, expect, app} = require('../common');
+const chaiHttp = require('chai-http');
+chai.use(chaiHttp);
+const chaiRequest = require('chai').request;
+// SHL: Please remove blank line between imports.
+
+const Unit = require('../../models/Unit')
+const { getUnitId } = require('../../util/readingsUtils');
+
+const validPayload = {
```

Comment:
Curious why this payload differs from other files. That could be fine but was it changed in certain ways to test other features?